src/csharp/Microsoft.Spark/Sql/Types/Date.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.Spark.Sql.Types
+{
+    /// <summary>
+    /// Represents Date containing year, month, and day.
+    /// </summary>
+    public class Date
+    {
+        private readonly DateTime _dateTime;
+        private static readonly DateTime s_unixTimeEpoch = new DateTime(1970, 1, 1);
+
+        /// <summary>
+        /// Constructor for Date class.
+        /// </summary>
+        /// <param name="dt">DateTime object</param>
+        public Date(DateTime dt)
```

Comment:
can we rename this to `dateTime` since we already have `_dateTime`?

Reply:
Updated.